projects/applicationinsights-angularplugin-js/.eslintrc.json

```diff
@@ -16,6 +16,36 @@
         "createDefaultProgram": true
       },
       "rules": {
+        "indent": ["error"],                                        // Enabling for auto fixing
+        "dot-notation": 0,
+        "quotes": ["error", "single"],                              // Enabled for auto fixing
```

Comment:
Should be double to match Application Insights

Reply:
Actually, ideally this should be the same as the Application Insights version -- for consistency